build_projects/dotnet-host-build/project.json

```diff
@@ -5,13 +5,13 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "NETStandard.Library": "1.6.0-rc3-24201-00",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24201-00",
-    "Microsoft.CSharp": "4.0.1-rc3-24201-00",
-    "System.Dynamic.Runtime": "4.0.11-rc3-24201-00",
-    "System.Reflection.Metadata": "1.3.0-rc3-24201-00",
-    "System.Runtime.Serialization.Primitives": "4.1.1-rc3-24201-00",
-    "System.Xml.XmlSerializer": "4.0.11-rc3-24201-00",
+    "NETStandard.Library": "1.6.0-rc3-24203-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24203-00",
```

Comment:
I'm working on getting the Ids updated to Runtime.CoreCLR, like we talked about in the other PR.

Yes, the UpdateDependencies script will need to be split, but we can't do that until corefx/release/1.0.0 builds are split.  @schellap is doing it for core-setup 'master' right now:  https://github.com/dotnet/core-setup/pull/82